connector/src/main/java/org/geysermc/connector/entity/living/animal/horse/LlamaEntity.java

```diff
@@ -47,6 +47,25 @@ public void updateBedrockMetadata(EntityMetadata entityMetadata, GeyserSession s
         if (entityMetadata.getId() == 19) {
             metadata.put(EntityData.STRENGTH, entityMetadata.getValue());
         }
+        // Color equipped on the llama
+        if (entityMetadata.getId() == 20) {
+            // Bedrock treats llama decoration as armor
+            MobArmorEquipmentPacket equipmentPacket = new MobArmorEquipmentPacket();
+            equipmentPacket.setRuntimeEntityId(getGeyserId());
+            // -1 means no armor
+            if ((int) entityMetadata.getValue() != -1) {
+                // The damage value is the dye color that Java sends us
+                // Always going to be a carpet so we can hardcode 171
+                // The int then short conversion is required or we get a ClassCastException
+                equipmentPacket.setChestplate(ItemData.of(171, (short)((int) entityMetadata.getValue()), 1));
+            } else equipmentPacket.setChestplate(ItemData.of(0, (short) 0, 0));
```

Comment:
```java
} else {
    equipmentPacket.setChestplate(ItemData.of(0, (short) 0, 0));
}
```

Reply:
Also, I'm pretty sure I created `ItemData.AIR`

Reply:
Yes, I noticed that. Will implement.